addons/account/models/account_incoterms.py

```diff
@@ -17,3 +17,9 @@ class AccountIncoterms(models.Model):
     active = fields.Boolean(
         'Active', default=True,
         help="By unchecking the active field, you may hide an INCOTERM you will not use.")
+
+    def name_get(self):
+        # Prefetch the fields used by the `name_get`, so `browse` doesn't fetch other fields
+        self.fetch(['name', 'code'])
```

Comment:
Is there a reason for this?
How is the prefetching an issue?

Reply:
Since the point was to have more or less the same thing as with the products displayed with their `default_code`, I looked into what had been made for that. Since there was that optimization made in `product.template`, I assumed it wasn't bad to add it here as well.
But maybe it's not relevant in this case?

Reply:
> `product.product` & `product.template` have a huge number of fields, and prefetching them just to compute the name is overkill.

https://github.com/odoo/odoo/commit/a91ffcf1a72f7d8759704405cd5026cf32c21a7f

It's not really the case for `account.incoterms`, is it? :slightly_smiling_face: 
It is better to avoid copy pasting code, so that other people are not tempted to copy paste it either without understanding the need for it.

Reply:
It is also less needed today thanks to `_name_search`, which doesn't do the `name_get`.
Because of this, you are very most likely only reading records with a limit anyways (usually only 6 records in the relational field widgets)